File: Baekjoon/2753 윤년.py
Year = int(input())

if (Year%4 == 0):
    if (Year%100 != 0):
        print(1)
    elif (Year%100 == 0):
        if (Year%400 == 0):
            print(1)
        else:
            print(0)
        
elif (Year%4 !=0):
    print(0)

else:
    print(0)

File: Baekjoon/2753_윤년.cpp
#include <iostream>
using namespace std;

int main () {
    int Year;
    cin >> Year;
    if (Year%4 == 0) {
        if (Year%100 != 0) {
            cout << 1 << endl;
        }
        else if (Year%100 == 0) {
            if (Year%400 == 0) {
                cout << 1 << endl;
            }
            else {
                cout << 0 << endl;
            }
        }
    }
    else if (Year%4 != 0)  {
        cout << 0 << endl;
    }
    else {
        cout << 0 << endl;
    }
}